modules/vsphere-problem-detector-metrics.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator-metrics_{context}"]
+= Metrics for the vSphere problem detector
```

Comment:
For consistency you could state the full name of the operator, capitalised in the titles in this assembly. For example:

` = Metrics for the vSphere Problem Detector Operator`

The same goes for the other titles, but I won't highlight them all in this review.